react/ProductList.tsx

```diff
@@ -54,14 +55,15 @@ const ProductGroup: StorefrontFunctionComponent<Props> = ({
   loading,
   onQuantityChange,
   onRemove,
+  renderOnView,
```

Comment:
React recommends defining defaultProps this way:
```suggestion
  renderOnView = true,
```